console/static/surveys/023.0205.json

```diff
@@ -8,21 +8,30 @@
     "instrument_id": "0205",
     "period": "1604"
   },
+  "submitted_at": "2016-03-12T10:39:40Z",
```

Comment:
The submitted at appears in all the other json example files. Do we not need this?